### PR TITLE
WEB-5: google maps set up

### PR DESCRIPTION
**Pull request by jennyckaplan**
This PR installs the necessary packages and sets up Google Maps SDK for iOS and Android.

### CODE REVIEW

**Comment by rlamsal1256** [comment]
can you provide the sources you used, please?

**Comment by jennyckaplan** [comment]
> can you provide the sources you used, please?

in the ticket

**Comment by rlamsal1256** [comment]
could you mention which commits/files were auto-generated or did you have to manually change all the ios and android specific files?

File: ios/Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,33 @@
+PODS:
```
Comment (rlamsal1256) [inline ios/Podfile.lock:1]:
```
Resolving dependencies of `Podfile`
    [!] CocoaPods could not find compatible versions for pod "GoogleMaps":
      In snapshot (Podfile.lock):
        GoogleMaps (= 3.5.0)
      In Podfile:
        google_maps_flutter (from `.symlinks/plugins/google_maps_flutter/ios`) was resolved to 0.0.1, which depends on
          GoogleMaps
    It seems like you've changed the constraints of dependency `GoogleMaps` inside your development pod `google_maps_flutter`.
    You should run `pod update GoogleMaps` to apply changes you've made.
```
got this error locally trying to run. did you have to do some setup work to make it run?

Comment (rlamsal1256) [reply ios/Podfile.lock:1]: running `pod update GoogleMaps` inside ios folder where the Podfile is located fixed the error and was able to run the app, but maybe worth adding this in the readme?

File: ios/Flutter/Release.xcconfig [hunk 2]
```diff
@@ -1 +1,3 @@
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"
 #include "Generated.xcconfig"
+#include "Pods/Target Support Files/Pods-Runner.profile.xcconfig"
```
Comment (jennyckaplan) [inline ios/Flutter/Release.xcconfig:3]: this line was manually added

File: ios/Podfile [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Using a CDN with CocoaPods 1.7.2 or later can save a lot of time on pod installation, but it's experimental rather than the default.
+# source 'https://cdn.cocoapods.org/'
+
+# Uncomment this line to define a global platform for your project
+platform :ios, '9.0'
```
Comment (jennyckaplan) [inline ios/Podfile:5]: this line was manually uncommented

**Comment by jennyckaplan** [comment]
> could you mention which commits/files were auto-generated or did you have to manually change all the ios and android specific files?

`add google_maps_flutter package`, `add Google Maps SDK for iOS`, `add Google Maps SDK for android` were manually added + see comments on other commits for manually added lines. the rest was autogenerated

File: ios/Flutter/Release.xcconfig (hunk 2 above)
Comment (rlamsal1256) [reply ios/Flutter/Release.xcconfig:3]: whats the reason for adding this if its commented?

Comment (jennyckaplan) [reply ios/Flutter/Release.xcconfig:3]: its not commented, that #include just means include this file

Comment (rlamsal1256) [reply ios/Flutter/Release.xcconfig:3]: oh lol thats confusing, my bad.

**Review by rlamsal1256 — APPROVED** [review]
Alt Text

**Pull request merged**